rpc_jobs/rpc_aio.yml

```diff
@@ -329,21 +329,20 @@
                   ]
                 aio_prepare.prepare()
                 deploy.deploy_sh(environment_vars: environment_vars)
-                tempest.tempest()
+                tempest.tempest_install()
```

Comment:
Having discussed this with @evrardjp I can see why the first tempest run could be removed. Firstly some of the base deploys are old versions of kilo that can't be modified and secondly the first deploy should be tested by other jobs.